src/components.js

```diff
@@ -125,23 +125,34 @@ export class Diagram extends Component {
     render() {
         let options = this.props.options == null ? '' : `[${this.props.options}]`
 
+        let cells = this.toArray().map(entries => entries.map(entry =>
+            entry == null ? ''
+            : [
+                (() => {
+                    let value = entry.node.props.value;
+                    let [w1, w2] = value != null
+                        && needWrapChars.some(c => value.includes(c))
+                        ? ['{', '}'] : ['', '']
+
+                    return value != null ? `${w1}${value}${w2}` : '{}'
+                })(),
+                ...entry.edges.map(e => renderEdge(e, this.props.co))
+            ].join(' ')
+        ))
+
+        if(this.props.align && cells.length > 0) {
+            for(let j in cells[1]) {
+                let maxLength = Math.max.apply(null, cells.map(entries => entries[j].length));
```

Comment:
Please remove all semicolons for code style consistency.

Reply:
Instead of `Math.max.apply(null, arr)` you can use the ES6 syntax `Math.max(...arr)`. It's more readable IMHO.